README.md

```diff
@@ -7,13 +7,43 @@ The build process takes around 3-4 hours with 8 CPU cores and 8 GiB RAM. A singl
 build requires ~70 GiB of disk space. Adding additional CPU cores speeds up the
 build time significantly.
 
-## Prerequisites
+## General requirements
 
 * Disk space and RAM
 
 We recommend a minimum of 8 GiB RAM and 150 GiB free disk space for building a single full
 image.
 
+* A functional Yocto build environment
+
+This can be achieved by either installing and configuring everything as
+documented in the section
+[Building your own build environment](#building-your-own-build-environment), or
+by starting from the
+[prebuild Yocto environment](https://github.com/bisdn/docker-yocto-builder) in a
+docker container, as [described below](#using-the-docker-yocto-builder).
+
+## Using the [docker-yocto-builder](https://github.com/bisdn/docker-yocto-builder)
+
+To ease the process of "getting started building BISDN Linux with Yocto", we
+have compiled all dependencies needed by Yocto itself, as well as all additional
+ones for specifically BISDN Linux, into a docker container. This container can
+be pulled directly from the GitHub container registry (ghcr)
```

Comment:
```suggestion
be pulled directly from the GitHub container registry ([ghcr](https://github.blog/2020-09-01-introducing-github-container-registry/))
```

Reply:
I don't think documentation (or an intro blog post about it) about ghcr belongs here.

Reply:
Maybe some link to what it is. I could not actually find anything satisfying so maybe it's better to just leave it like this.

Reply:
Not sure what you mean here?

Reply:
updated

Reply:
Good solution